src/main/scala/cognite/spark/v1/DataSetsRelation.scala

```diff
@@ -1,16 +1,15 @@
 package cognite.spark.v1
 
 import cats.effect.IO
-import cats.implicits._
 import cognite.spark.v1.PushdownUtilities.{executeFilterOnePartition, pushdownToFilters, timeRange}
 import cognite.spark.v1.SparkSchemaHelper._
 import com.cognite.sdk.scala.common.WithId
 import com.cognite.sdk.scala.v1.resources.DataSets
 import com.cognite.sdk.scala.v1.{DataSet, DataSetCreate, DataSetFilter, DataSetUpdate, GenericClient}
 import io.scalaland.chimney.Transformer
-import org.apache.spark.sql.{Row, SQLContext}
 import org.apache.spark.sql.sources.{Filter, InsertableRelation}
-import org.apache.spark.sql.types.{DataTypes, StructType}
+import org.apache.spark.sql.types._
```

Comment:
..but this one I disagree with (it's nice to be able to see a list of what's in use from a library from here). Nit-picky, I know :)